concepts/regular-expressions/about.md

```diff
@@ -0,0 +1,124 @@
+# About 
+
+Package [regexp][package-regexp] offers support for regular expressions in Go.
+
+## Syntax 
+
+The [syntax][regexp-syntax] of the regular expressions accepted is the same general syntax used by Perl, Python, and other languages. 
+
+All characters are UTF-8-encoded code points. 
+Following `utf8.DecodeRune`, each byte of an invalid UTF-8 sequence is treated as if it encoded `utf8.RuneError (U+FFFD)`.
+
+It is convenient to write regular expressions as [raw string literals][raw-string-literals] enclosed by backticks.
+This avoids having to quote the backslashes.
+
+## Type `RegExp`
+
+The package defines a type `Regexp` for compiled regular expressions.
+Function `regexp.Compile` compiles a string into a regular expression.
+The function returns `nil` and an error if compilation failed:
+
+```go
+re, err := regexp.Compile(`(a|b)+`)
+fmt.Println(re, err) // => (a|b)+ <nil>
+re, err = regexp.Compile(`a|b)+`)
+fmt.Println(re, err) // => <nil> error parsing regexp: unexpected ): `a|b)+`
+```
+
+Function `MustCompile` is like `Compile` but panics if the expression cannot be parsed.
+It simplifies safe initialization of global variables holding compiled regular expressions.
```

Comment:
I think we can expand on this a bit more here and compare the two. For instance, say something along the lines of `MustCompile` being easier to use because we don't need to handle the error, but also mention that it must be used with extreme caution and only when we know for sure the pattern does compile, otherwise the program will panic.